src/app/Console/Commands/CreateUser.php

```diff
@@ -44,11 +45,15 @@ public function handle()
             $password = $this->secret('Password');
         }
 
+        if(! $this->option('encrypt')) {
+          $password = bcrypt($password);
```

Comment:
Oh how I hate that helper - wouldn't it be better to use the `Hash` facade? Especially since Argon2 is now supported...

Reply:
My reasoning for using the bcrypt helper is that by default that is what vanilla Laravel and Backpack uses by default within the `RegisterController`. Was trying to stick with what is currently used.